source/clear-linux/tutorials/aws-web/aws-web.rst

```diff
@@ -0,0 +1,266 @@
+      :scale: 50 %
+      :alt: Step 7: Review the Instance Launch
+
+      Figure 6: :guilabel:`Step 7: Review the Instance Launch`
+
+   #. Select the :guilabel:`Launch` button. A dialog box appears, as shown in
+      figure 7, asking you to
+      :guilabel:`Select an existing key pair or create a new pair`.
+
+      .. figure:: figures/aws-web-7.png
+         :scale: 50 %
+         :alt: Select an existing key pair or create a new pair
+
+         figure 7: :guilabel:`Select an existing key pair or create a new pair`
+
+      #. Select the :guilabel:`Create a new key pair` option.
+      #. For the :guilabel:`Key pair name`, enter `AWSClearTestKey`.
+      #. Select the :guilabel:`Download Key Pair` button to download the
+         :file:`AWSClearTestKey.pem` to your browser's defined
+         :file:`Downloads` directory.
+      #. When the file finishes downloading, select the
+         :guilabel:`Launch Instances` button to proceed to the
+         :guilabel:`Launch Status` menu shown in figure 8.
+
+         .. figure:: figures/aws-web-8.png
+            :scale: 50 %
+            :alt: Launch Status
+
+            figure 8: :guilabel:`Launch Status`
+
+      #. Once the :guilabel:`Launch Status` page changes to what is shown in
+         figure 9, select the :guilabel:`View Instances` button to view your
+         :guilabel:`Instances` dashboard.
+
+         .. figure:: figures/aws-web-9.png
+            :scale: 50 %
+            :alt: View Instance
+
+            figure 9: :guilabel:`View Instance`
+
+Connect to your Clear Linux OS Basic instance
+*********************************************
+
+Your :guilabel:`Instances` Dashboard is shown in figure 10 with the new |CL|
+OS basic instance already selected and in the running state. If there are
+other instances available, they are also listed but not selected.
+
+.. figure:: figures/aws-web-10.png
+   :scale: 50 %
+   :alt: Instance Dashboard
+
+   figure 10: :guilabel:`Instance Dashboard`
+
+#. To connect to your running instance, click the :guilabel:`Connect` button
+   located at the top of your dashboard. This brings up the pop-up dialog
+   box shown in figure 11 describing how to connect to your running instance. 
+
+.. figure:: figures/aws-web-11.png
+   :scale: 50 %
+   :alt: Connect to Your Instance
+
+   figure 11: :guilabel:`Connect to Your Instance`
+
+#. Open a terminal on your system. You should be in your :file:`$HOME`
+   directory.
+
+#. Copy the previously downloaded keyfile from the :file:`Downloads`
+   directory.
+
+   .. code-block:: console
+
+      cp Downloads/AWSClearTestKey.pem .
+
+#. Change the attributes of the :file:`AWSClearTestKey.pem` using the
+   :command:`chmod` command as instructed in the dialog box shown in figure
+   11.
+
+   .. code-block:: console
+
+      chmod 400 AWSClearTestKey.pem
+
+#. Copy the text highlighted in the :guilabel:`Example:` section that is
+   shown in figure 11. Paste the copied text into your terminal and add
```

Comment:
Link back to figure 11. If in doubt on how to do it, let me know.

Reply:
not to sure how to do this

Reply:
I can help. I'll schedule some time.